.config/cargo_spellcheck.dic

```diff
@@ -1,75 +1,45 @@
-80
+40
 
 WebAssembly
 Wasm
-Polkadot
 validator
-LLVM
-JIT
-rustup
-miri
-clippy
-hunspell
 ERC
 fuzzer
-1A
-2A
-1B
-2B
 SHA2
 BLAKE2
 KECCAK
 ABI
 FFI
 UI
 crypto
-Dev
 deploy
 layed
 dispatchable
-backend
 SRML
-Params
 encodable
 decodable
 spreaded
 callee
 prefilled
-|
 defrag
 defragments
-defragmentation
-prev
 growable
-th
 codegen
```

Comment:
ideally we'd use "code generation" instead but I am not sure about this since "codegen" is so commonly used in compiler speak.

Reply:
I think it's fine to leave it, since we also have the `ink_lang_codegen` and when using the term we often kinda refer to that part of the codebase.